src/repos/NotificationRepo.ts

```diff
@@ -60,7 +60,52 @@ const notify = async (articleIDs: string[]): Promise<void> => {
   });
 };
 
+const sendWeeklyDebriefNotification = async (user: User): Promise<void> => {
+  // Get device token of user
+  const { deviceToken } = user;
+
+  const notifTitle = 'Your Weekly Debrief is Ready';
```

Comment:
do you think there would be value in putting these const titles, bodies, messages, and options into a separate file to make it easy to find them when we need to change them if ever? do we plan to have any other kinds of notifications?

Reply:
As of now the only notifications with constant titles is the weekly debrief. I believe magazines might have notifications (maybe not since they are semesterly), but they wouldn't have constant titles however.